internal/commands/secrets/create_inputs.go

```diff
@@ -0,0 +1,52 @@
+package secrets
+
+import (
+	"github.com/10gen/realm-cli/internal/app"
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/terminal"
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	flagName      = "secret-name"
+	flagNameShort = "n"
+	flagNameUsage = `the name of the secret to add to your Realm App`
+
+	flagValue      = "secret-value"
+	flagValueShort = "v"
+	flagValueUsage = `the value of the secret to add to your Realm App`
+
+	createInputFieldSecretName  = "Name"
```

Comment:
[meganit] these field consts actually should be able to be lowercased (but still snakeCased) and survey recognizes them